app/api/resources/user.py

```diff
@@ -241,8 +245,12 @@ def get(cls):
         location, occupation, organization, interests, skills, need_mentoring,
         available_to_mentor. The current user's details are not returned.
         """
+
+        page = request.args.get("page", default=1, type=int)
+        per_page = request.args.get("per_page", default=10, type=int)
```

Comment:
Could we define this default numbers somewhere else in the code, for example on config.py?

Reply:
atm we put our constant for users in UserDAO (app/api/dao/user.py). Can we put the default for page at the same place for consistency, @isabelcosta ?

<img width="540" alt="Screen Shot 2020-05-06 at 8 19 30 am" src="https://user-images.githubusercontent.com/29667122/81121607-7b8b6c80-8f72-11ea-8c48-672d04f921da.png">

Reply:
i am doubting my decision of putting it there :P @mtreacy002 you have a great eye for detail 👏 So yes, for now @brianguo98 could put there for consistency, and then we can change that :)